iac-extensions/terraform/src/test/resources/checks/GCP/ComputeInstanceSshKeysCheck/ssh_keys.tf

```diff
@@ -0,0 +1,63 @@
+resource "google_compute_instance" "example1" { # Noncompliant {{Omitting metadata.block-project-ssh-keys enables project-wide SSH keys. Make sure it is safe here.}}
+#        ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Typically the _omitting_ message only references the missing block or attribute nur a chain of missing elements.